test/helpers/publishers_helper_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class PublishersHelperTest < ActionView::TestCase
+  test "should render brave publisher id as a link" do
+    publisher = publishers(:default)
+    assert_dom_equal %{<a href="http://#{publisher.brave_publisher_id}">#{publisher.brave_publisher_id}</a>},
```

Comment:
just a minor note, wish we could know when it's https:// vs not so we could show https:// when appropriate

Reply:
The protocol will sometimes be known (as we're fixing in #53). We could capture it when it's provided. Is that something that is worth the effort, given it won't be 100% (or would require the entry of a full url)? Even with the protocol we might be missing the other parts of a complete and resolvable URL like the hostname, path, and parameters.

Reply:
you are correct it's probably not worth the effort